### DISCUSSION TITLE
iOS/Safari remote play stopped working

### DISCUSSION THREAD

**Discussion by UncleLeoTheDad** [redphx/better-xcloud#418] [General]
I'm on version 4.6.2 currently. When I load up the xcloud page everything looks fine. I go to launch a remote play session. I briefly see the rocket animation and then the screen goes black. After 30 seconds or so a brief message is shown, but it's too fast for me to read or capture. I am then brought back to the landing page.  I've tried:
- Redownloading the user script
- Clearing my caches
- Rebooting my iPhone
- Hard-rebooting my iPhone

Any other ideas?  Remote play works fine on my android tablet and my MacBook.

**Comment by UncleLeoTheDad** [comment]
I should also mention that cloud streaming works fine. So the problem is only with the remote play.

I've also tried rebooting my Xbox.

**Comment by redphx** [comment]
Try older versions. If it still happens then it must be your network.

**Comment by UncleLeoTheDad** [comment]
Hmmm but it did work for a very long time and it didn’t break via an
upgrade.

That said,  I will try old versions as you suggested.

Network problem: wouldn’t that impact all my other devices as well?  They
work fine.

**Comment by UncleLeoTheDad** [comment]
@redphx i tried multiple versions, all the way back to 4.0. Same result. For network issues, can you give me a hint where to look? I am only seeing this on iOS safari. It works fine on android. Thanks!

**Comment by UncleLeoTheDad** [comment]
Oh I should also mention that the official Xbox app for remote play works fine.
